modules/fedimint-lnv2-client/src/lib.rs

```diff
@@ -380,6 +380,39 @@ impl LightningClientModule {
         Ok(operation_id)
     }
 
+    async fn get_next_operation_id(
+        &self,
+        invoice: &Bolt11Invoice,
+    ) -> Result<OperationId, SendPaymentError> {
+        for payment_attempt in 0..u64::MAX {
```

Comment:
Very nit: `0..` is shorter and suggest to me that "it will never reach the end" better.